### PR TITLE
Projects api cleanup 119119925

### PR DESCRIPTION
**Pull request by jjdelc**

### CODE REVIEW

File: source/includes/_endpoint-projects.md [hunk 1]
```diff
@@ -19,17 +20,17 @@ GET /projects/ HTTP/1.1
 ```json
 {
   "element": "shoji:catalog",
-  "self": "http:\/\/local.crunch.io:8080\/api\/projects\/",
+  "self": "http:\/\/beta.crunch.io\/api\/projects\/",
```
Comment (nealrichardson) [inline source/includes/_endpoint-projects.md]: What's the deal with the \/\/? ASCII art? Please remove.

File: source/includes/_endpoint-projects.md [hunk 2]
```diff
@@ -269,14 +284,32 @@ PATCH /projects/abcd/members/ HTTP/1.1
...
+###### Sending notifications
+
+The users invited to a project can be both existing Crunch.io users or new
+users that don't have a user account associated with the email.
+
+If desired, the API can send automated email notifications to the involved users
+indicating that they now belong to the project.
+
+It is necessary to add the `send_notification` boolean key on the index PATCHed
+to command the API to send these emails. Else, no notification will be sent.
```
Comment (nealrichardson) [inline source/includes/_endpoint-projects.md:340]: What about `url_base`?

File: source/includes/_endpoint-projects.md [hunk 3]
```diff
@@ -225,32 +228,35 @@ GET /projects/abcd/members/ HTTP/1.1
 ```json
 {
   "element": "shoji:catalog",
-  "self": "http:\/\/local.crunch.io:8080\/api\/projects\/6c01\/members\/",
+  "self": "http:\/\/beta.crunch.io\/api\/projects\/6c01\/members\/",
   "index": {
-    "http:\/\/local.crunch.io:8080\/api\/users\/00002\/": {
+    "http:\/\/beta.crunch.io\/api\/users\/00002\/": {
       "name": "Jean-Luc Picard",
       "email": "captain@crunch.io"
```
Comment (nealrichardson) [inline source/includes/_endpoint-projects.md]: Why doesn't captain have permissions?

File: source/includes/_endpoint-projects.md [hunk 4]
```diff
@@ -225,32 +228,35 @@ GET /projects/abcd/members/ HTTP/1.1
...
     },
-    "http:\/\/local.crunch.io:8080\/api\/users\/00005\/": {
+    "http:\/\/beta.crunch.io\/api\/users\/00005\/": {
       "name": "William Riker",
       "email": "firstofficer@crunch.io",
       "permissions": {
-        "edit: false,
+        "edit": false,
         "view": true
```
Comment (nealrichardson) [inline source/includes/_endpoint-projects.md:266]: Please enumerate the allowed and required permissions. This still isn't clear enough for me to write an API client.

File: source/includes/_endpoint-projects.md [hunk 5]
```diff
@@ -269,14 +312,41 @@ PATCH /projects/abcd/members/ HTTP/1.1
...
+to command the API to send these emails. Else, no notification will be sent.
+
+When sending notifications, it is necessary for the client to include a 
+`url_base` key as well that includes a string template that should point to a
+client location where the password resetting should happen for brand new users.
+
+The server will replace the `${token}` part of the string with the generated
+token and will be included on the notification email as a link for the invited 
+user to configure their account in order to use the app.
+
```
Comment (nealrichardson) [inline source/includes/_endpoint-projects.md:349]: Please add a link back to the sharing/permissions docs since this is the same idea as what's there.

Comment (jjdelc) [inline source/includes/_endpoint-projects.md:349]: done

File: source/includes/_endpoint-projects.md (hunk 2 above)
Comment (jjdelc) [inline source/includes/_endpoint-projects.md:340]: added

File: source/includes/_endpoint-projects.md (hunk 4 above)
Comment (jjdelc) [inline source/includes/_endpoint-projects.md:266]: Added more info about the permissions

**Pull request merged**